### DISCUSSION TITLE
can't load no_images.svg

### DISCUSSION THREAD

**Discussion by Ryanonly** [LycheeOrg/Lychee#2313] [Q&A] [answered]
When I was transferring images, I found that one of my options did not display the image
like this
![Snipaste_2024-03-09_21-33-15](https://github.com/LycheeOrg/Lychee/assets/155050703/115c243b-836b-4ca9-97b3-cd1d3adb236b)
my errors.log says:
![Snipaste_2024-03-09_21-34-49](https://github.com/LycheeOrg/Lychee/assets/155050703/cfea5d0b-c3ed-4449-b2d7-9894e206e979)

But the image on the homepage can be loaded normally
![Snipaste_2024-03-09_21-40-28](https://github.com/LycheeOrg/Lychee/assets/155050703/a2f991ea-3da3-4c3b-a02a-ddb0c47277b1)
same svg in the options：
![Snipaste_2024-03-09_21-38-34](https://github.com/LycheeOrg/Lychee/assets/155050703/1e32479c-80df-4f26-8b14-351deca5c34e)

System Information
    ------------------
        Lychee Version (release):                5.1.2
        DB Version:                              5.1.2
        
        composer install:                        --no-dev
        APP_ENV:                                 production
        APP_DEBUG:                               false
        APP_URL:                                 set
        APP_DIR:                                 default
        LOG_VIEWER_ENABLED:                      true
        LIVEWIRE_ENABLED:                        true
        
        System:                                  Linux
        PHP Version:                             8.2.16
        PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
        Timezone:                                Asia/Shanghai
        Max uploaded file size:                  30M
        Max post size:                           100M
        Livewire chunk size:                     12.00 MB
        Max execution time:                      200
        MySQL Version:                           8.0.36-0ubuntu0.22.04.1
        
        exec() Available:                        yes
        Imagick Available:                       1
        Imagick Enabled:                         1
        Imagick Version:                         1691
        GD Version:                              2.3.3
        Number of foreign key:                   12 found.

Config Information
    ------------------
        version:                                 050102
        check_for_updates:                       0
        sorting_photos_col:                      taken_at
        sorting_photos_order:                    ASC
        sorting_albums_col:                      max_taken_at
        sorting_albums_order:                    ASC
        imagick:                                 1
        skip_duplicates:                         0
        small_max_width:                         0
        small_max_height:                        360
        medium_max_width:                        1920
        medium_max_height:                       1080
        lang:                                    en
        image_overlay_type:                      desc
        default_license:                         none
        compression_quality:                     90
        grants_full_photo_access:                1
        delete_imported:                         0
        mod_frame_enabled:                       1
        mod_frame_refresh:                       30
        thumb_2x:                                1
        small_2x:                                1
        medium_2x:                               1
        landing_page_enable:                     0
        site_owner:                              John Smith
        landing_title:                           John Smith
        landing_subtitle:                        Cats, Dogs & Humans Photography
        sm_facebook_url:                         https://www.facebook.com/JohnSmith
        sm_flickr_url:                           https://www.flickr.com/JohnSmith
        sm_twitter_url:                          https://www.twitter.com/JohnSmith
        sm_instagram_url:                        https://instagram.com/JohnSmith
        sm_youtube_url:                          https://www.youtube.com/JohnSmith
        landing_background:                      dist/cat.webp
        site_title:                              Lychee v5
        footer_show_copyright:                   1
        site_copyright_begin:                    2019
        site_copyright_end:                      2019
        footer_additional_text:                  
        footer_show_social_media:                0
        search_public:                           0
        SL_enable:                               0
        SL_for_admin:                            0
        recent_age:                              1
        grants_download:                         0
        photos_wraparound:                       1
        map_display:                             1
        zip64:                                   1
        map_display_public:                      1
        map_provider:                            Wikimedia
        force_32bit_ids:                         0
        map_include_subalbums:                   1
        update_check_every_days:                 3
        has_exiftool:                            0
        share_button_visible:                    0
        import_via_symlink:                      0
        has_ffmpeg:                              0
        location_decoding:                       1
        location_decoding_timeout:               30
        location_show:                           0
        location_show_public:                    1
        rss_enable:                              0
        rss_recent_days:                         7
        rss_max_items:                           100
        prefer_available_xmp_metadata:           0
        editor_enabled:                          1
        lossless_optimization:                   0
        swipe_tolerance_x:                       150
        swipe_tolerance_y:                       250
        local_takestamp_video_formats:           .avi|.mov
        log_max_num_line:                        1000
        unlock_password_photos_with_url_param:   0
        nsfw_visible:                            1
        nsfw_blur:                               0
        nsfw_warning:                            0
        nsfw_warning_admin:                      0
        nsfw_banner_override:                    
        map_display_direction:                   1
        album_subtitle_type:                     oldstyle
        upload_processing_limit:                 4
        new_photos_notification:                 0
        legacy_id_redirection:                   1
        zip_deflate_level:                       6
        SA_enabled:                              1
        default_album_protection:                1
        album_decoration:                        layers
        album_decoration_orientation:            row
        allow_username_change:                   1
        auto_fix_orientation:                    1
        use_job_queues:                          0
        random_album_id:                         starred
        use_last_modified_date_when_no_exif_date: 0
        ffmpeg_path:                             
        ffprobe_path:                            
        layout:                                  justified
        date_format_photo_thumb:                 M j, Y, g:i:s A e
        date_format_photo_overlay:               M j, Y, g:i:s A e
        date_format_sidebar_uploaded:            M j, Y, g:i:s A e
        date_format_sidebar_taken_at:            M j, Y, g:i:s A e
        date_format_hero_min_max:                F Y
        date_format_hero_created_at:             M j, Y, g:i:s A T
        date_format_album_thumb:                 M Y
        upload_chunk_size:                       0
        nsfw_banner_blur_backdrop:               0
        search_pagination_limit:                 1000
        search_minimum_length_required:          4
        photo_layout_justified_row_height:       320
        photo_layout_masonry_column_width:       300
        photo_layout_grid_column_width:          250
        photo_layout_square_column_width:        200
        photo_layout_gap:                        12
        display_thumb_album_overlay:             always
        display_thumb_photo_overlay:             hover
        default_album_thumb_aspect_ratio:        1/1
        use_album_compact_header:                0
        login_button_position:                   left
        back_button_enabled:                     0
        back_button_text:                        Return to Home
        back_button_url:                         /

I want some help, thank you all！

**Comment by Ryanonly** [comment]
![Screenshot_2024-03-11-13-33-03-541_com android browser](https://github.com/LycheeOrg/Lychee/assets/155050703/a65e1ed7-afa9-414e-bd65-6ad786b48170)
and got this today

**Comment by ildyria** [comment, marked as answer]
This is most likely v5, and this is fixed in v6. :)
